config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  root 'messages#index'
+  resources :messages
+
+  namespace :api do
+    namespace :v1 do
+      resources :messages, only: %i[index]
+    end
+  end
```

Comment:
- [ ] The react app is unable to fetch data from this backend due to CORS policy. You can fix this by installing the gem 'rack-cors' and adding a config/initializers/cors.rb file and allow requests from all sites. You can visit the documentation here: [rack-cors gem documentation](https://github.com/cyu/rack-cors).

  ![fkfe](https://user-images.githubusercontent.com/36542986/196937215-c8af9a1d-4a24-473d-a566-50d4f3ccb7f5.jpg)